listenbrainz/webserver/static/js/src/user-feed/UserFeed.tsx

```diff
@@ -405,8 +407,122 @@ export default class UserFeedPage extends React.Component<
     }
   };
 
+  hideFeedEvent = async (event: TimelineEvent) => {
+    const { currentUser, APIService } = this.context;
+    const { newAlert } = this.props;
+    const { events } = this.state;
+
+    try {
+      const status = await APIService.hideFeedEvent(
+        event.event_type,
+        currentUser.name,
+        currentUser.auth_token as string,
+        event.id!
+      );
+
+      if (status === 200) {
+        newAlert("success", "", <>Successfully hidden</>);
```

Comment:
Not sure we want to show an alert toast, because the visual feedback  of the event being hidden should be enough